### ISSUE TITLE
Standardise on lexicon for js pos taggers?

### ISSUE THREAD

**Issue by giorgio79** [spencermountain/compromise#524]
https://github.com/finnlp/en-pos has a nice one at

https://github.com/finnlp/en-lexicon#readme

**Comment by spencermountain** [comment]
yeah, i'd love to see more attention regarding standards for different lexicons, that was the dream of the plugin scheme, that people would collaberate, and build-up eachothers lexicons, so that there are well-built one-liners for supporting jamaican slang, or whatever. This would be great, but it just hasn't happened.

The current compromise lexicon is so custom, because it's composed in large-part, on the fly, by conjugation/inflection, etc. - all for reduction of file-size. Also, it tries to match as much as possible using regexs, with the suffix-list and things. The idea was that only the words that slip through the cracks actually need to be listed-out. - that was the idea, at least..

**Comment by giorgio79** [comment]
Found one more https://github.com/dariusk/corpora/ used by  https://github.com/dariusk/pos-js :)  I guess editing large files is a pain for most hence the lack of coordination. VS Code handles large files very well though :P
